test/Test/Toml/Printer/Golden.hs

```diff
@@ -0,0 +1,49 @@
+-- | This module contains golden tests for @Toml.Printer@.
+
+module Test.Toml.Printer.Golden
+       ( test_prettyGolden
+       ) where
+
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.Silver (goldenVsAction)
+import Toml (AnyValue (..), TOML (..), Value (..))
+import Toml.PrefixTree ((<|))
+import Toml.Printer (PrintOptions (..), pretty, prettyOptions)
+
+import qualified Data.HashMap.Strict as HashMap
+import qualified Toml.PrefixTree as PrefixTree
+
+example :: TOML
+example = mempty
+    { tomlPairs  = HashMap.fromList
```

Comment:
Actually, we have eDSL implemented by @vrom911 for creating TOML data types. It was designed specially for such cases. See the following module:

* https://github.com/kowainik/tomland/blob/master/src/Toml/Edsl.hs

And here is one usage example:

* https://github.com/kowainik/tomland/blob/3187fed8dc353b68553f229b5dfb76fdc4e5e9f0/examples/Playground.hs#L81-L95

So you can write this test in more convenient way. And we also will test our eDSL as well 🙂

Reply:
:+1: